tests/php/tests/includes/test_class.wp-job-manager-email-notifications.php

```diff
@@ -425,10 +425,20 @@ public function test_add_email_settings() {
 			$email_key                = $email_keys[ $key ];
 			$email_settings           = call_user_func( [ $email_class, 'get_setting_fields' ] );
 			$email_is_default_enabled = call_user_func( [ $email_class, 'is_default_enabled' ] );
-			$defaults                 = [
-				'enabled'    => $email_is_default_enabled ? '1' : '0',
-				'plain_text' => '0',
-			];
+			if ( 'job_manager_email_employer_expiring_job' === $setting['name'] ) {
+				$defaults                 = [
```

Comment:
You can write this like this:
```
			$defaults                 = [
				'enabled'    => $email_is_default_enabled ? '1' : '0',
				'plain_text' => '0',
			];
			$email_settings_count = count( $email_settings ) + 1;

			if ( in_array( $setting['name'], [ 'job_manager_email_admin_updated_job', 'job_manager_email_admin_new_job', 'job_manager_email_admin_expiring_job'] ) ) {
				$email_settings_count++;
				$defaults['email_to'] = '';
			}
```

This makes it a bit more robust as it covers future changes of the settings in the test and makes it a bit easier to read as an 'if' is always easier to read than an 'if/else'